llarp/dht/messages/findintro.hpp

```diff
@@ -35,14 +35,14 @@ namespace llarp
       }
 
       FindIntroMessage(uint64_t txid, const llarp::service::Address& addr,
-                       bool iterate = true)
+                       bool iterate = false)
           : IMessage({}), S(addr), T(txid)
       {
         N.Zero();
         if(iterate)
           R = 0;
         else
-          R = 1;
+          R = 3;
```

Comment:
I don't understand this change (I'm assuming R is the recursion level).  If client C connects to router X and asks for an introset, X knows that A is the closest router to the requested introset, so connects to A to get the introset then passes it back to C.  But this change here looks like it's going to ask A to get it recursively, and that doesn't make sense to me.